pkg/storage/common/common.go

```diff
@@ -1116,16 +1116,19 @@ type smTask struct {
 	imgStore storageTypes.ImageStore
 	metrics  monitoring.MetricServer
 	repo     string
+	log      zlog.Logger
 }
 
 func NewStorageMetricsTask(imgStore storageTypes.ImageStore, metrics monitoring.MetricServer, repo string,
+	log zlog.Logger,
 ) *smTask {
-	return &smTask{imgStore, metrics, repo}
+	return &smTask{imgStore, metrics, repo, log}
 }
 
 func (smt *smTask) DoWork(ctx context.Context) error {
 	// run task
 	monitoring.SetStorageUsage(smt.metrics, smt.imgStore.RootDir(), smt.repo)
+	smt.log.Debug().Msg("monitoring: computed storage usage for repo " + smt.repo)
```

Comment:
.Str("repo", smt.repo).Msg("...")

Reply:
Agree, but for testing purposes where we wait for some specific string messages in the logs before proceeding further and asserting, we need the message in that form unless we will start using some json go libraries for better log parsing